Packs/Thycotic/Integrations/Thycotic/Thycotic.yml

```diff
@@ -0,0 +1,1096 @@
+    - auto: PREDEFINED
+      default: false
+      description: isGroupOwnerUpdate
+      isArray: false
+      name: isgroupownerupdate
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether the user is locked out
+      isArray: false
+      name: isLockedout
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: Number of login failures
+      isArray: false
+      name: loginfailures
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether OATH two-factor authentication is enabled
+      isArray: false
+      name: oathtwofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: Password
+      isArray: false
+      name: password
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether RADIUS two-factor authentication is enabled
+      isArray: false
+      name: radiustwofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: RADIUS username
+      isArray: false
+      name: radiususername
+      required: false
+      secret: false
+    - default: false
+      description: timeOptionId (integer)
+      isArray: false
+      name: timeoptionid
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether two-factor authentication is enabled
+      isArray: false
+      name: twofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    deprecated: false
+    description: Update a single user by ID
+    execution: false
+    name: thycotic-user-update
+    outputs:
+    - contextPath: Thycotic.User.Update
+      description: User object
+      type: String
+  - arguments:
+    - default: false
+      description: User ID
+      isArray: false
+      name: id
+      required: true
+      secret: false
+    deprecated: false
+    description: Delete user by ID
+    execution: false
+    name: thycotic-user-delete
+    outputs:
+    - contextPath: Thycotic.User.Delete
+      description: User object
+      type: String
+  - arguments:
+    - default: false
+      description: Secret ID
+      isArray: false
+      name: secret_id
+      required: true
+      secret: false
+    - default: false
+      description: Password
+      isArray: false
+      name: newpassword
+      required: true
+      secret: false
+    deprecated: false
+    description: Remote Password Changing
+    execution: false
+    name: thycotic-secret-rpc-changepassword
+    outputs:
+    - contextPath: Thycotic.Secret.ChangePassword
+      description: Secret summary
```

Comment:
I don't understand this description, what can be found here?

Reply:
returns a secret object according to API.
from API:
`

Secret summary

active: boolean

    Whether the secret is active
autoChangeEnabled: boolean

    Indicates whether or not this Secret an auto changing password
checkedOut: boolean

    Whether the secret is currently checked out
checkOutEnabled: boolean

    Indicates whether or not checkout is enabled for the Secret
createDate: string (date-time)

    When the Secret was created
daysUntilExpiration: integer (int32)

    How many days until this Secret expires
doubleLockEnabled: boolean

    Indicates whether or not DoubleLock is enabled for this password
extendedFields: object[]

    Any requested extended fields from a lookup request
    ISecretSummaryExtendedField
folderId: integer (int32)

    Containing folder ID
hidePassword: boolean

    Indicates if the launcher password is set to be hidden
id: integer (int32)

    Secret ID
inheritsPermissions: boolean

    Indicates if this Secret inherits permissions from its folder
isOutOfSync: boolean

    Out of sync indicates that a Password is setup for autochange and has failed its last password change attempt or has exceeded the maximum RPC attempts
isRestricted: boolean

    Whether the secret is restricted
lastAccessed: string (date-time)

    When the Secret was last viewed, only populated when scope is Recent
lastHeartBeatStatus: HeartbeatStatus

    Current status of heartbeat
lastPasswordChangeAttempt: string (date-time)

    Time of most recent password change attempt
name: string

    Secret name
outOfSyncReason: string

    Reason message if the secret is out of sync
requiresApproval: boolean

    Indicates if this Secret requires approval
requiresComment: boolean

    Indicates if this Secret requires comment
responseCodes: string[]

    ResponseCodes
    string
secretTemplateId: integer (int32)

    Secret template ID
secretTemplateName: string

    Name of secret template
siteId: integer (int32)

    SiteId`

Reply:
So there's an entire dictionary of values under this context path?